hazelcast/src/main/java/com/hazelcast/instance/GroupProperties.java

```diff
@@ -57,11 +57,32 @@
      * <p/>
      * The performance monitor logs all metrics into the log file.
      */
-    public static final String PROP_PERFORMANCE_MONITORING_ENABLED = "hazelcast.performance.monitoring.enabled";
+    public static final String PROP_PERFORMANCE_MONITOR_ENABLED = "hazelcast.performance.monitoring.enabled";
+
+    /**
+     * The delay in seconds between monitor of the performance.
```

Comment:
is it a delay or an interval of measurement cycles? perhaps a native English speaker could have look here. @Danny-Hazelcast  ?